### PR TITLE
New styling for Veteran Name and ID

### PR DESCRIPTION
**Pull request by Sjones352**
Connects https://github.com/department-of-veterans-affairs/caseflow/issues/2216

## AC
1. Verify that the veteran name and ID number currently to the the right of the application name 'eFolder Express' are removed.
1. Verify that the user name displays in title case, 17px, color #5B616B
1. Verify that the Veteran Name is left aligned and displays in all caps, 22px, color #323A44
1. Verify that the Veteran ID is displayed in color: #5B616A, style: bold, 17px, inside a button to the left of the 'copy' icon (also color: #5B616A). Button fill color: #FFFFFF, with a border: 2px, color: #5B616A.
1. Verify that there are 30px above the Vet ID button, and 30 px below the vet ID button

efolder veteran

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -67,7 +67,7 @@
           <a class="cf-header-link" href="/">
             <h1 class="cf-logo"><span class="cf-logo-image"></span>eFolder Express</h1>
           </a>
-          <h2 id="page-title" class="cf-application-title"><%= yield :page_title %></h2>
+          <h2 id="page-title" class="cf-application-title"></h2>
```
Comment (mmlumba) [inline app/views/layouts/application.html.erb]: If you remove this you remove the page title for all pages (we want to keep the title of the stats page up). Instead, remove the first line of `app/views/downloads/show.html.erb` so the veteran name doesn't appear anymore.

File: app/views/downloads/_confirm.html.erb [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+
 <% if @download.fetching_manifest? %>
   <script>
     // Set up automatic download status updates
     window.DownloadStatus.init(<%= @download.id %>, "<%= @download.status %>");
   </script>
 <% end %>
 
-<% fetch_button_text = "Start retrieving eFolder" %>
+<%= render "user_header" %>
```
Comment (mmlumba) [inline app/views/downloads/_confirm.html.erb]: Depending on what @lakohl says, you might need to put`<%= render "user_header" %>` in `_progress.html.erb` too. Right now this header doesn't show when the eFolder is being retrieved or the eFolder is finished retrieving.

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -67,7 +67,7 @@
           <a class="cf-header-link" href="/">
             <h1 class="cf-logo"><span class="cf-logo-image"></span>eFolder Express</h1>
           </a>
-          <h2 id="page-title" class="cf-application-title"><%= yield :page_title %></h2>
+          <h2 id="page-title" class="cf-application-title"><%=yield :page_title %></h2>
```
Comment (mmlumba) [inline app/views/layouts/application.html.erb]: nit: Leave the space between the `=` and `yield`

File: app/views/downloads/_user_header.html.erb [hunk 4]
```diff
@@ -1 +1,14 @@
-<h1><%= @download.veteran_name %> <span class='ee-subheading'>(<%=h @download.file_number %>)</span></h1>
+
+<div class="cf-efolder-header cf-app-segment">
+    <h3 class="cf-push-left cf-name-header cf-efolder-name" id="correspondent-name">
```
Comment (mmlumba) [inline app/views/downloads/_user_header.html.erb]: Is this ID being styled in some other file? If not can you delete it?

File: app/views/downloads/_user_header.html.erb [hunk 5]
```diff
@@ -1 +1,14 @@
...
+      <%=@download.veteran_name  %>
```
Comment (mmlumba) [inline app/views/downloads/_user_header.html.erb]: Ruby convention: There should always be a space after `=` in a render tag

File: app/views/downloads/_confirm.html.erb (hunk 2 above)
Comment (Sjones352) [inline app/views/downloads/_confirm.html.erb]: Fixed

File: app/views/downloads/_user_header.html.erb (hunk 5 above)
Comment (Sjones352) [inline app/views/downloads/_user_header.html.erb]: Fixed

File: app/views/downloads/_user_header.html.erb (hunk 4 above)
Comment (Sjones352) [inline app/views/downloads/_user_header.html.erb]: Fixed

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (Sjones352) [inline app/views/layouts/application.html.erb]: Fixed

**Pull request merged**